File: result/translate/starchat/java2python/python/MINIMUM_INSERTIONS_SORT_ARRAY.py
def f_gold(arr, N):
        lis = [1] * N
        for i in range(N):
            for j in range(i):
                if arr[i] >= arr[j] and lis[i] < lis[j] + 1:
                    lis[i] = lis[j] + 1
        max = 0
        for i in range(N):
            if max < lis[i]:
                max = lis[i]
        return N - max

File: result/translate/starchat/java2cpp/c++/MINIMUM_INSERTIONS_SORT_ARRAY.cpp
int f_gold ( vector<int> arr ) {
    	vector<int> lis ( arr.size(), 1 );
    	for ( int i = 0; i < arr.size(); i++ ) {
    		for ( int j = 0; j < i; j++ ) {
    			if ( arr[i] >= arr[j] && lis[i] < lis[j] + 1 ) {
    				lis[i] = lis[j] + 1;
    			}
    		}
    	}
    	int max = 0;
    	for ( int i = 0; i < arr.size(); i++ ) {
    		if ( max < lis[i] ) {
    			max = lis[i];
    		}
    	}
    	return arr.size() - max;
    }